search.go

```diff
@@ -7,10 +7,10 @@ import (
 
 // Search gets Spotify Catalog information about albums, artists, playlists, tracks, shows or episodes that match a
 // keyword string.
-func (a *API) Search(q string, limit int) (*Paging, error) {
+func (a *API) Search(q, searchType  string, limit int) (*Paging, error) {
```

Comment:
Just call it `type` to match the variable name here: https://developer.spotify.com/documentation/web-api/reference/#endpoint-search

```suggestion
func (a *API) Search(q, type string, limit int) (*Paging, error) {
```

Reply:
<img width="459" alt="Screenshot 2021-07-18 at 13 51 12" src="https://user-images.githubusercontent.com/49570057/126065942-8904f685-9a83-4ef9-a35b-a1783fc01fd5.png">
'type' seems to be a controlled keyword and can't be used as a param (Which is why I put it as searchType in the beginning) - I can try something else if you have a suggestion?

Reply:
Oops. `searchType` is ok then.